src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java

```diff
@@ -10,25 +10,24 @@
 /**
  * Created by nikodem on 27.07.15.
  */
+// TODO: What is this class about?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)